### PR TITLE
Switch to the only prepublish script

### PR DESCRIPTION
**Pull request by usulpro**
Issue: All packages use different versions of prepublish scripts. Plus some of them `shell` scripts which have bad compatibility with windows

## What I did

- Added the only instance of prepublish script to
`./scripts/prepublish.js`
it's the shell js version wich allready was used in [`storybook-ui`](https://github.com/storybooks/storybook-ui/blob/master/.scripts/prepublish.js)

- Changed in `scripts` of each `package.json` to
`"prepublish": "node ../../scripts/prepublish.js"`

- Removed previos versions of prepublish scripts from packages

## How to test

1. clone this repo
2. run `yarn`
it should install all dependencies without any errors and buid `dist`'s for all packages

### CODE REVIEW

File: scripts/prepublish.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+const shell = require('shelljs');
+const chalk = require('chalk');
+const babel = ['node_modules', '.bin', 'babel'].join(path.sep);
```
Comment (shilman) [inline scripts/prepublish.js]: Use `path.join`?

**Review by shilman — APPROVED** [review]
Great change! See minor comment above.

File: scripts/prepublish.js (hunk 1 above)
Comment (UsulPro) [inline scripts/prepublish.js]: sure!

**Comment by UsulPro** [comment]
Thanks, @shilman! 
I added `--copy-files` flag as well

**Comment by UsulPro** [comment]
So if nobody objects I will merge this?

**Comment by ndelangen** [comment]
@UsulPro Are you working on getting this merged?

**Comment by UsulPro** [comment]
Yes @ndelangen I'll resolve it

**Pull request merged**